FHEM/00_SIGNALduino.pm

```diff
@@ -5087,10 +5086,11 @@ When set to 1, the internal "RAWMSG" will not be updated with the received messa
 	<a name="SIGNALduinointernals"></a>
 	<b>Internals</b>
 	<ul>
-		<li><b>LASTDMSGID</b>: Hier wird die zuletzt dispatchte Protocol ID angezeigt.</li>
```

Comment:
Was ist der Grund den Eintrag zu entfernen? Haben wir das entfernt aber vergessen in der commandref aufzuräumen?

Reply:
Die dort benannten Internals habe ich nur sortiert so wie diese auch in den Internals sind.
Die Zeile wurde "entfernt" weil diese verschoben wurde.